### PR TITLE
.coafile: Add HTMLLintBear

### PR DESCRIPTION
**Pull request by sks444**
This adds the HTMLLintBear and its suggested changes.

Closes https://github.com/coala/community/issues/117

### CODE REVIEW

File: .coafile [hunk 1]
```diff
@@ -31,6 +31,12 @@ max_line_length = 120
 [all.links]
 bears = InvalidLinkBear
 
+[html]
+bears = HTMLLintBear
+files = **.html
+htmllint_ignore = http_equiv, entities, optional_tag, concerns_separation, names, protocol
```
Comment (nalinbhardwaj) [inline .coafile]: Why are so many rules being ignored?

`http_equiv` seems fixable by replacing https://github.com/coala/community/blob/master/templates/activity.html#L13 by `<meta charset="utf-8">`

`entities` only triggers on https://github.com/coala/community/blob/master/templates/index.html#L11 which to me seems like a bug? I'm not sure why there is a carriage return there, maybe quotes were intended originally?

`protocol` is also fixable across the files.

I haven't manually checked others, but try to not ignore the rules.

**Comment by sks444** [comment]
Thanks, @nalinbhardwaj, for the review.
Can you take a look again :)

File: templates/index.html [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+  <ul>
     <li><a href="/gci/">Google Code-in</a>
     <li><a href="/activity/">GitHub activity</a>
     <li><a href="/twitter/">Twitter Feed</a>
     <li><a href="/info.txt">Build info</a>
     <li><a href="/log/">Logs</a>
     <li><a href="/contributors/">Contributors data</a>
-    <li><a title="List of all the issues on organization&#13;&#10;main repository on which assignee&#13;&#10;has not shown any activity&#13;&#10;for more than 2 months."
-           href="/static/inactive-issues.json">Inactive Issues</a>
+    <li><a title="List of all the issues on organization main repository on which assignee has not shown any activity for more than 2 months."
```
Comment (nalinbhardwaj) [inline templates/index.html]: I think it should be `organization's`?

**Comment by sks444** [comment]
@nalinbhardwaj, there seems to be some problem on breaking the commits into two. That bear will run on unchanged files and the ci will not pass.
 
What to you mean by this line?
>If you're going to keep a single commit, make it multi

**Comment by nalinbhardwaj** [comment]
@sks444 : it should pass if you make the HTML changes before the coala config changes I think?

> What do you mean by this line?

By that, I just meant replacing `.coafile:` in the shortlog by `multi:` prefix, since the change spans multiple files.

**Comment by sks444** [comment]
>it should pass if you make the HTML changes before the coala config changes I think?

Yeah, but doesn't sound good, right? At first, we are adding the bear then we are finding what changes to make. 

Going for the `multi:` then. :)

**Comment by sks444** [comment]
@nalinbhardwaj, looks good?

File: templates/activity.html [hunk 3]
```diff
@@ -1,41 +1,39 @@
...
-    }
-    </style>
-    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
-</head>
-
-<body>
+        }
+      </style>
+      <meta charset="utf-8">
+      <meta charset="utf-8" content="text/html">
```
Comment (blazeu) [inline templates/activity.html]: This is probably not needed, 2 meta charset?

Comment (sks444) [reply templates/activity.html]: I think I should remove the line-12 meta tag?

File: templates/index.html [hunk 4]
```diff
@@ -1,15 +1,15 @@
 <title>Community website</title>
```
Comment (jayvdb) [inline templates/index.html]: why no `<html lang="en">` required?

Comment (sks444) [reply templates/index.html]: No idea, why it was not added at the first place. I will add it 👍

**Comment by sks444** [comment]
>split into two commits; one for tiding templates/: .. and another for .coafile: ...

Alright👍
Actually, we thought that it wouldn't be good to commit the changes suggested by bear before it.

**Comment by jayvdb** [comment]
ack https://github.com/coala/community/pull/122/commits/812f613cd8d5b572f28d71316aebbecbd0e13bed https://github.com/coala/community/pull/122/commits/9261c9f6d0777a150e3ee69b146e4d7f52a4eb82

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**